cedar-policy/src/ffi/convert.rs

```diff
@@ -0,0 +1,396 @@
+/*
+ * Copyright Cedar Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//! JSON FFI entry points for converting between JSON and Cedar formats. The
+//! Cedar Wasm conversion functions are generated from the functions in this
+//! file.
+
+use super::utils::JsonValueWithNoDuplicateKeys;
+use super::{DetailedError, Schema, Template};
+use serde::{Deserialize, Serialize};
+#[cfg(feature = "wasm")]
+use wasm_bindgen::prelude::wasm_bindgen;
+
+#[cfg(feature = "wasm")]
+extern crate tsify;
+
+/// Return the Cedar (textual) representation of a policy or template.
+///
+/// Note that the [`Template`] type can represent both static policies and
+/// policy templates.
```

Comment:
Good note, I had forgotten this was the case in the FFI.  It's not true of `cedar_policy::Template`.

Reply:
Having `Policy` in the FFI support both static policies and templates might be additionally confusing because `Policy` in `cedar_policy` represents both static policies and template-linked policies, but not templates

Reply:
Yuck. I'll leave it as-is then.